src/uu/stty/src/stty.rs

```diff
@@ -438,6 +449,18 @@ fn cc_to_index(option: &str) -> Option<SpecialCharacterIndices> {
     None
 }
 
+fn string_to_combo(arg: &str) -> Option<&str> {
+    let negate = arg.starts_with('-');
```

Comment:
I wouldn't use a verb for a boolean variable. Maybe something like `is_negated` would be better?